py/server/deephaven/csv.py

```diff
@@ -74,19 +85,19 @@ def read(
             for column_name, column_type in header.items():
                 csv_specs_builder.putParserForName(column_name, parser_map[column_type])
 
-        csv_specs = (
-            csv_specs_builder.hasHeaderRow(not headless)
-            .skipRows(skip_rows)
-            .numRows(num_rows)
-            .ignoreEmptyLines(ignore_empty_lines)
-            .allowMissingColumns(allow_missing_columns)
-            .ignoreExcessColumns(ignore_excess_columns)
-            .delimiter(ord(delimiter))
-            .quote(ord(quote))
-            .ignoreSurroundingSpaces(ignore_surrounding_spaces)
-            .trim(trim)
-            .build()
-        )
+        csv_specs = (csv_specs_builder.hasHeaderRow(not headless)
+                     .skipHeaderRows(header_row)
```

Comment:
I would add some kind of clarification about what happens if you say headless and specify a header_row that isn't 0. Also, why headless and not header_less, or has_header defaulting to true?

Reply:
Not sure, I don't like neg/neg either, could be from a prototype Devin did?

Reply:
I do like `has_header` more than the negation, but I don't _think_ it was something I did... might have been carried over from previous CSV stuff?